### PR TITLE
Update react native examples

### PR DESCRIPTION
**Pull request by SlicedSilver**
#### Update example checklist

- [x] **Any** commit does not contain the charting library's code
- [x] Addresses an existing issue: #247 #267 #297 #314 #315
- `N/A` ~The example is added to README.md~
- [x] The example is self-sufficient and contains only necessary files/changes

Updates the `react-native` example.

### CODE REVIEW

File: react-native/README.md [hunk 1]
```diff
@@ -1,22 +1,27 @@
-# TradingView Charting Library React-Native. Integration Example
+# React Native Example
 
 ## How to start - Android
 
 1. Setup the development environment. [React Native Docs](https://reactnative.dev/docs/environment-setup)
+   1. Ensure that you follow all the steps for installing dependencies such as node, watchman, Java development kit, and Android development environment
+   2. You can return this guide once you reach the 'Creating a new application' section.
```
Comment (romfrancois) [inline react-native/README.md:7]: typo - return **_to_** this

**Review by Sl-violet — APPROVED** [review]

**Review by romfrancois — APPROVED** [review]

**Comment by tayyabrajpoot12** [comment]
@SlicedSilver I download your repo for react native app but in android this is showing blank , Im unable to understand your above discussion , can you please clear it in details.

**Comment by SlicedSilver** [comment]
Did you run all these steps? https://github.com/tradingview/charting-library-examples/tree/master/react-native#how-to-start---android

Including copying the library files into the specified folder.

**Comment by tayyabrajpoot12** [comment]
> Did you run all these steps? https://github.com/tradingview/charting-library-examples/tree/master/react-native#how-to-start---android
> 
> Including copying the library files into the specified folder.

I am getting 404 where I have to copy the files ,  and now as mentioned in the docs make a request on https://www.tradingview.com/HTML5-stock-forex-bitcoin-charting-library/ 
can you tell me what I have to do next  ? means I have to need subscription plans ?

**Comment by edew** [comment]
You need to request access to the library:

1. Go to https://www.tradingview.com/advanced-charts/
2. Click "Get the library"
3. Follow the instructions

**Pull request merged**